.gitignore

```diff
@@ -137,3 +137,7 @@ dmypy.json
 
 # IntelliJ
 .idea/
+
+# VScode
```

Comment:
Could you submit this separately? Changing the local Git repository configuration does not seem to be directly related to the navigator settings.

Reply:
+1. Removing .vscode from the commit.

Reply:
FYI I see a new article like this every year: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/